client/store/userchallenge.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+export const SET_USER_CHALLENGE = Symbol('set user challenge');
+
+export const setUserchallenge = userchallenge => ({ type: SET_USER_CHALLENGE, userchallenge });
+
+export const userchallenge = (state = [], action) => {
+  switch (action.type) {
+    case SET_USER_CHALLENGE:
+      return action.userchallenge;
+    default:
+      return state;
+  }
+};
+
+export const postUserchallenge = userAnswer => dispatch => {
```

Comment:
Need `challengeId` parameter for line 19
```suggestion
export const postUserchallenge = (userAnswer, challengeId) => dispatch => {
```

Reply:
I agree, I will update the index.js and store.js as well